packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/legacy/webview_flutter_test.dart

```diff
@@ -757,7 +757,10 @@ Future<void> main() async {
       isPaused = await controller.runJavascriptReturningResult('isPaused();');
       expect(isPaused, _webviewBool(false));
     });
-  });
+  },
+      // OGG playback is not supported on macOS, so the test data would need
+      // to be changed to support macOS.
+      skip: Platform.isMacOS);
```

Comment:
nit: Is this indentation correct?

Reply:
I don't know why the formatter does this; it's something to do with having a comment inline. We hit this with `skip` frequently, unfortunately.

The only alternative I can get the formatter to do is by adding a comma after `isMacOS`, but then it indents the entire test more, which is annoying for consistency with other tests.